pkg/registry/apis/provisioning/resources/parser.go

```diff
@@ -203,6 +204,39 @@ func (r *Parser) Parse(ctx context.Context, info *repository.FileInfo, validate
 		return parsed, nil
 	}
 
+	// FIXME: This is actually not okay in a "parse" function
```

Comment:
@MissingRoberto  will take over this fix: https://raintank-corp.slack.com/archives/C07V91MA04B/p1743672651576749?thread_ts=1743607042.093559&cid=C07V91MA04B